New documentation structure and content (#195)
liteflow-labs/liteflow-js: docs/pages/products/nft-data.mdx

---
title: 'NFT Data'
---

import { Card } from '../../components/Card'
import { Cards, Steps } from 'nextra-theme-docs'

# NFT Data

> NFT Data solution serves as a comprehensive toolkit for aggregating, synchronizing, and retrieving NFT data across diverse blockchains. Designed to simplify the complexities of blockchain data, our platform ensures near-instantaneous indexing of NFTs, readily available through our APIs, SDKs, or templates.

Reviewer: ```suggestion
The NFT Data solution serves as a comprehensive toolkit for aggregating, synchronizing, and retrieving NFT data across diverse blockchains. Designed to simplify the complexities of blockchain data, our product facilitates near-instantaneous indexing of NFTs, making data accessible through a robust set of APIs, SDKs, and customizable templates.
```

> Our array of indexers interacts seamlessly with various supported blockchains, capturing detailed NFT data, from ownerships and transfers to trades and beyond. We normalize and meticulously structure this information, enabling swift and efficient querying via our [GraphQL API](/tools/graphql-api) or our [SDKs](/tools/sdk-core).

Reviewer: ```suggestion
Our indexing solutions are designed to interact with a range of blockchains, ensuring comprehensive data collection including ownership details, transfer histories, and market activities. By standardizing this data, we empower you with efficient query capabilities using our [GraphQL API](/tools/graphql-api) or directly within our [SDKs](/tools/sdk-core).
```

## Features

<Cards num={3}>

>   <Card title="Multi-Blockchain Support">

Reviewer: ```suggestion
  <Card title="Multi-Blockchain Compatibility">
```

>     Facilitate access to NFT data across a variety of blockchains.

Reviewer: ```suggestion
    Unify NFT data management across various  blockchains.
```

  </Card>
  <Card title="Dedicated API & SDK">

>     Streamline access to all synchronized NFT data.

Reviewer: ```suggestion
    Provide streamlined, direct access to synchronized NFT data.
```

  </Card>

>   <Card title="Ease of Querying">

Reviewer: ```suggestion
  <Card title="Efficient Querying">
```

>     Simplify data retrieval with straightforward GraphQL queries.

Reviewer: ```suggestion
    Employ user-friendly GraphQL queries for hassle-free data retrieval.
```

  </Card>
  <Card title="Selective Indexing">

>     Choose specific collections to include in your dataset.

Reviewer: ```suggestion
    Tailor your indexing preferences by choosing specific collections to include in your dataset.
```

  </Card>
  <Card title="Real-Time Indexing">

>     Acquire updated data promptly as transactions are confirmed.

Reviewer: ```suggestion
    Receive immediate indexing updates following transaction confirmations.
```

  </Card>

>   <Card title="Historical Data">

Reviewer: ```suggestion
  <Card title="Historical Records">
```

    Access a comprehensive activity history of your NFTs.
  </Card>
</Cards>

## How to Use It

<Steps>

> ### Index Collections

Reviewer: ```suggestion
### Collection Indexing
```

> Log in to the Liteflow [dashboard](/tools/dashboard), and choose to import an external NFT collection or create a new one tailored to your business needs.

Reviewer: ```suggestion
Log in to the Liteflow [dashboard](/tools/dashboard), and either choose to import an existing NFT collection or create a new one to match your business specifications.
```

### Access Methods

Retrieve synchronized NFT data using the [marketplace template](/templates/marketplace), [SDK](/tools/sdk-core), or [GraphQL API](/tools/graphql-api), according to your specific requirements.

> ### Data Visualization

Reviewer: ```suggestion
### Data Oversight
```

> Directly manage and monitor all NFT-related information through the [dashboard](/tools/dashboard), ensuring comprehensive oversight.

Reviewer: ```suggestion
Monitor and manage NFT data through the [dashboard](/tools/dashboard) for a complete view of your NFT landscape. 
```
